lib/client-transport.js

```diff
@@ -180,8 +180,9 @@ TAmqpTransport.prototype.write = function (buf) {
 TAmqpTransport.prototype.flushQueue = function (msg) {
   var self = this;
   return new Promise(function (resolve, reject) {
+    var tempQueueName = self.options.tempQueuePrefix + '.gen-' + uuid.v1();
```

Comment:
is there a reason why we would still do the `.gen` with uuid?

Reply:
Same reason ampq did it by default. Just to denote, to a human, that this queue was generated.